tests/testparse.cpp

```diff
@@ -87,15 +88,16 @@ void TestParse::testParseCSV()
 void TestParse::testParseDivingLog()
 {
 	// Parsing of DivingLog import from SQLite database
-	sqlite3 *handle;
+	sqlite3 *handle = NULL;
+	ScopedShutdownHelper scoped_shutdown( [&](){
+						sqlite3_close(handle);
+					});
```

Comment:
I do like lambdas but I agree that here it is not necessary and self contained test cases is much better. When I see something I don't understand or agree with (not self contained test cases) I often think there must be a reason I don't know or did not see for doing it this way. I therefore preferred not to change that, now that I have your "go ahead" I'll proceed. (I go dive now but will deal with this later today...)
thanks for the review